plugins/out_kinesis_firehose/firehose.h

```diff
@@ -41,7 +41,7 @@ struct flush {
     size_t data_size;
 
     /* log records- each of these has a pointer to their message in tmp_buf */
-    struct event *events;
+    struct firehose_event *events;
```

Comment:
is this a breaking change? Do others reference these h files?

Reply:
This is just an internal data structure used in kinesis_firehose Output plugin which is used for the log events related to Firehose. It is not referenced anywhere else in the code.

Reply:
Ok as long as we are confident nobody builds off of this since its a leaf this is fine.